dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -1580,6 +1581,12 @@ SearchHits cachedIndexSearch(final SearchRequest searchRequest) {
         
     }
 
+    private void rebuildRestHighLevelClientIfNeeded(Exception e) {
+        if(e.getMessage().contains("reactor status: STOPPED")) {
```

Comment:
I would null check `e.getMessage()` before the contains.

Also, should we `if(e instanceof IllegalStateException)`?

Reply:
maybe as a separate catch?

Reply:
Done